### DISCUSSION TITLE
FormDropdown should be used for production app

### DISCUSSION THREAD

**Discussion by shabbir-dhangot** [microsoft/fluentui#27014] [Q&A] [answered]
Hello,

Currently we are facing issue with the FormDropdown component for covering test cases. While looking around for solution we read a comment where it says `FormDropdown is an experiment checked in several months ago, but no longer maintained by the original author. We expect to remove it in the future.` 

https://github.com/microsoft/fluentui/issues/6077#issuecomment-416340989

According to this I am in confusion that should I use FormDropdown for production app or not? @micahgodbolt 

Regards,
Shabbir

**Comment by micahgodbolt** [comment, marked as answer]
Seems there was a bit of confusion on that old issue. The author stated that:

> For clarification FormDropdown is my wrapper around Dropdown. The issue is with the Dropdown component.

There is a FormDropdown in the current Northstar package. This package is meant for production, though it is in the process of being replaced with the new Fluent UI v9. 

I wouldn't suggest taking a new dependency on this control, but if you are using it in production, there shouldn't be an issue.
